src/test/java/lambda/part2/exercise/Exercise1.java

```diff
@@ -0,0 +1,62 @@
+package lambda.part2.exercise;
+
+import java.util.function.BiPredicate;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+@SuppressWarnings({"unused", "ConstantConditions"})
+public class Exercise1 {
+
+    @Test
+    public void ageExtractorFromPersonUsingMethodReference() {
+        Person person = new Person("Иван", "Мельников", 33);
+
+        Function<Person, Integer> ageExtractor = x -> x.getAge();
```

Comment:
I'll be glad if you change this expression-lambda to **method reference** ```Person::getAge```

Reply:
Lambda changed to method reference